packages/plugin-analytic-google/src/GoogleAnalytic.js

```diff
@@ -95,11 +89,7 @@ export default class GoogleAnalytic extends AbstractAnalytic {
   _configuration() {
     const clientWindow = this._window.getWindow();
 
-    if (
-      !clientWindow.ga ||
-      typeof this._window.getWindow().ga !== 'function' ||
-      this.isEnabled()
-    ) {
+    if (!clientWindow.ga || typeof this._window.getWindow().ga !== 'function') {
```

Comment:
why did you remove isEnabled()?

Reply:
Because the `_configuration` method won't be even executed if `this.isEnabled()` returns `true`.
See https://github.com/seznam/IMA.js-plugins/pull/6/files#diff-5a1a4fc5265fa9626cc10c3db3756d1aR86